CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
+* Add a notes field to trade, order and product types ([#1563](https://github.com/finos/FDC3/pull/1563))
```

Comment:
The PR in question only added the notes field to the trade type.

Another open PR #1503 goes further and adds notes to Order and Product. It also created the RFQ type and was trying to define Product properly (which we probably won't merge).